miden/tests/integration/exec_iters.rs

```diff
@@ -0,0 +1,155 @@
+use super::build_test;
+use processor::VmState;
+use vm_core::{utils::ToElements, Felt};
+
+// EXEC ITER TESTS
+// =================================================================
+#[test]
+fn test_exec_iter() {
+    let source = "proc.foo.1 push.local.0 end begin popw.mem.1 push.17 exec.foo end";
+    let mut init_stack: Vec<u64> = Vec::new();
+    (1..=16).for_each(|i| {
+        init_stack.push(i);
+    });
+    let test = build_test!(source, &init_stack.clone());
+    let traces = test.execute_iter();
+    let fmp = Felt::new(1073741824);
```

Comment:
nit: `Felt::new(2.pow(30))` is probably better from readability standpoint.

Reply:
fixed